docs/en/observability/monitor-infra/view-infrastructure-metrics.asciidoc

```diff
@@ -30,7 +34,166 @@ Alternatively, you can change this map view to a table view.
 
 Without leaving the *Inventory* page, you can view enhanced metrics relating to each host
 running in your infrastructure. On the waffle map, select the host to display the host metrics
-overlay. For more information, see <<enhanced-host-details,*Host details*>>.
+overlay.
+
+The host details overlay contains the following tabs:
+
+// This is collapsed by default
+[%collapsible]
+.*Metrics*
+====
+
+[role="screenshot"]
+image::images/metrics-overlay.png[Host metrics]
+
+The *Metrics* tab displays CPU, load, memory, and network metrics relating to the host, along with the log rate
+and any custom metric that you have defined. You can change the time range to view metrics over the last 15 minutes,
+hour, 3 hours, 24 hours, or over the previous seven days. You can also hover over a specific time period on a chart
+to compare the various metrics at that given time.
+
+A chart is also displayed for each <<custom-metrics,custom metric>> that you
+have added and defined on the **Inventory** page.
+
+====
+
+[%collapsible]
+.*Logs*
+====
+
+[role="screenshot"]
+image::images/logs-overlay.png[Host logs]
+
+The *Logs* tab displays logs relating to the host that you have selected. By default, the logs tab displays the following columns. 
+
+|=== 
+
+| *Timestamp* | The timestamp of the log entry from the `timestamp` field. 
+
+| *Message* | The message extracted from the document.
+The content of this field depends on the type of log message.
+If no special log message type is detected, the {ecs-ref}/ecs-base.html[Elastic Common Schema (ECS)]
+base field, `message`, is used.
+
+|=== 
+
+You can customize the logs view by adding a column for an arbitrary field you would like
+to filter by. For more information, refer to <<customize-stream-page,Customize Stream>>.
+To view the logs in the {logs-app} for a detailed analysis, click *Open in Logs*.
+====
+
+[%collapsible]
+.*Processes*
+====
+
+[role="screenshot"]
+image::images/processes-overlay.png[Host processes]
+
+//REVIEWERS: Lising all the source fields here feels like it could incur a lot
```

Comment:
Reviewers: Please respond to this question about whether we should continue to list all these source fields.